test/transformers/test_cross_entropy.py

```diff
@@ -687,6 +709,41 @@ def test_correctness_with_z_loss_with_other_params_once(
     )
 
 
+@pytest.mark.parametrize(
+    "B, T, V",
+    [
+        (2, 4096, 32000),  # llama2, mistral
+        # # weird shapes
+        (3, 423, 32000),
+    ],
+)
+@pytest.mark.parametrize("weight", [0.5, 0.1])
```

Comment:
I think this param can be removed, since the weight tensor is being generated from rand.